README.md

```diff
@@ -53,22 +42,6 @@ So that I can stay constantly tapped in to the shouty box of Chitter
 I want to receive an email if I am tagged in a Peep
 ```
 
```

Comment:
Would it be possible to have a gif or a video walkthrough of how the project works